Workbooks/SecurityStatus.json

```diff
@@ -0,0 +1,582 @@
+{
```

Comment:
Hi,
There is a readme file with instructions on how to contribute workbook templates to Sentinel.
This PR lacks some important steps that are explained there.
You can find it in the workbooks directory in this repository.
Here is a link  - [workbooks documentation](https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/README.md)